setup.sh

```diff
@@ -0,0 +1,4 @@
+./gradlew build
```

Comment:
I've made this script which is already on master. Once you merge master in this will conflict.